common/utilities/timer.h

```diff
@@ -0,0 +1,112 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+// Helper classes to keep track of time
+#pragma once
+
+#include <chrono>
+namespace time_utilities
+{
+    using clock = std::chrono::steady_clock;
+
+    class stopwatch
+    {
+    public:
+        stopwatch()
+        {
+            _start = clock::now();
+        }
+
+        // Reset the stopwatch time
+        void reset(clock::time_point start_time = clock::now()) const 
+        { 
+            _start = start_time; 
+        }
+
+        // Get elapsed in milliseconds since timer creation
+        double elapsed_ms() const
```

Comment:
Why is this needed?

Reply:
It was an existing function that get called from several places,
It hides a very long code line.
I prefer keeping it and not handle each place that called it if you don't mind.